src/transport_ofi.c

```diff
@@ -605,6 +609,17 @@ static inline int publish_av_info(struct fabric_info *info)
     char   epname[128];
     size_t epnamelen = sizeof(epname);
 
+#ifdef USE_ON_NODE_COMMS
+    gethostname(myephostname, (EPHOSTNAMELEN - 1));
```

Comment:
Should we check the return value from gethostname?  The manpage says:

> Upon successful completion, the value 0 is returned; otherwise the value -1 is returned and the global variable errno is set to indicate the error.